charts/kyverno-policies/templates/pod-sec-ctxt-vcs.yaml

```diff
@@ -0,0 +1,39 @@
+{{- $name := "pod-sec-ctxt-vcs" }}
```

Comment:
@rbak-hpe you'll want to pair this with a validation rule to keep the same convention as other policies here. Also, @mitchty has a PR in flight that re-arranges the directory structure a bit, FYI: https://github.com/Cray-HPE/cray-kyverno-policies/pull/4.